.travis.yml

```diff
@@ -3,6 +3,7 @@ language: python
 env:
   - TENV=py27
   - TENV=py36
+  - TENV=itest
 python:
   - 2.7
```

Comment:
Can't we get rid of this part of the matrix?

If we are running 3.6 in the dh-virtualenv, I don't see any reason to test against 2.7 at all.